sdcm/cluster.py

```diff
@@ -4763,6 +4763,13 @@ def get_db_nodes_cpu_mode(self):
         self.log.info("DB nodes CPU modes: %s", results)
         return results
 
+    def wait_node_fully_start(self, node, timeout=3600):
```

Comment:
it probably works, but if this function is placed inside the "node" object itself, then you don't have to pass a `node` as a parameter, and then you can just do:

```
self.log......
self.wait_db_up(timeout)
self.log.....
self.wait_jmx_up()
self.log.......
```